sdk/identity/azure-identity/setup.py

```diff
@@ -60,7 +60,7 @@
     install_requires=[
         "azure-core<2.0.0,>=1.11.0",
         "cryptography>=2.5",
-        "msal<2.0.0,>=1.12.0",
+        "msal<2.0.0,>=1.20.0b1",
```

Comment:
Should we use [1.20.0](https://pypi.org/project/msal/1.20.0/)?

Reply:
> Should we use [1.20.0](https://pypi.org/project/msal/1.20.0/)?

You are absolutely right!